provider/cmd/pulumi-resource-cloud-toolkit-aws/databases/mysqlArgs.ts

```diff
@@ -0,0 +1,227 @@
+import defaultsDeep from "lodash.defaultsdeep";
+import * as pulumi from "@pulumi/pulumi";
+
+/**
+* Arguments to deploy a database instance
+*/
+export interface MysqlArgs {
+/**
+ * Instance type to run the database instance
+ */
+  instance?: pulumi.Input<string>;
+
+/**
+ * Version for database instance
+*/
+  version: pulumi.Input<string>;
+
+/**
+* Configuration parameters for the database instance
+*/
+  database: MysqlDatabaseArgs;
+
+/**
+* Storage configuration parameters for the database instance
+*/
+  storage?: MysqlStorageArgs;
+
+/**
+* Network configuration parameters for the database instance
+*/
+  networking?: MysqlNetworkingArgs;
+
+/**
+* Backup configuration parameters for the database instance
+*/
+  backup?: MysqlBackupArgs;
+}
+
+/**
+* Backup configuration parameters for the database instance
```

Comment:
Wrong descriptions

Reply:
What do you mean by "wrong descriptions"? Descriptions should not be there or description is not appropiate for the code purpose.?

Reply:
Now it should be fixed